src/app/views/components/table/table.component.html

```diff
@@ -65,9 +65,9 @@ <h3 class="forge-typography--subtitle1">
         <forge-table select="false" #selectAllTemplateTable
           [select]="true"
           [multiselect]="true"
-          [selectCheckboxAlignment]="'right'"
```

Comment:
This one is interesting, not sure if the latest version of TS would address it, but because the type of this property is a non-const enum, I can't just assign a string literal, even if it's a valid member of the enum.  I think we may want to change how the property is defined in Forge to address this.

Reply:
Yeah I generally use a `type` for these things nowadays...

Reply:
I can also just specify the type on the component itself as `` selectCheckboxAlignment: `${CellAlign}` `` which effectively converts it to a union type, but still leave the enum in play elsewhere so I don't have to refactor all the usages of it.  But as a pattern going forward, we may want to prefer such types over enums for this reason.